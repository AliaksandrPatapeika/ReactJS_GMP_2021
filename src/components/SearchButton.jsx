import '../less/components/SearchButton.less';

import React from 'react';

const SearchButton = () => <button type="button" className="searchButton">SEARCH</button>;

export default SearchButton;
