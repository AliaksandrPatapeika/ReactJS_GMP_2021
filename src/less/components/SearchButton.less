@import "../variables";

.searchButton {
	color: @white;
	background-color: @red;
	padding: 16px 77px;
	font-size: 22px;
	border-radius: 4px;
	cursor: pointer;
	outline: none;
	font-family: @NetflixSans-Regular;
}