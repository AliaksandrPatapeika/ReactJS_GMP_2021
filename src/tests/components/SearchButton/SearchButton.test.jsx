import React from 'react';
import renderer from 'react-test-renderer';

import SearchButton from '../../../components/SearchButton';

describe('SearchButton', () => {
  test('renders correctly', () => {
    const searchButton = renderer
      .create(<SearchButton />)
      .toJSON();

    expect(searchButton).toMatchSnapshot();
  });
});
