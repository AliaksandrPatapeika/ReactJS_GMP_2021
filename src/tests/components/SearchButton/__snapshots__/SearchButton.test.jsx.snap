// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchButton renders correctly 1`] = `
<button
  className="searchButton"
  type="button"
>
  SEARCH
</button>
`;
